wp-content/themes/core/components/search_form/Search_Form_Controller.php

```diff
@@ -60,15 +60,6 @@ public function get_label(): string {
 		return esc_html( $this->label );
 	}
 
-	public function get_clear_button_args(): array {
```

Comment:
@dpellenwood Have the SQ1 designs been updated to reflect all search forms removing the clear button? It's been nice to have it included here as all of the redesigns T2 has done in the past couple years have used it. I don't care either way, just want to make sure design isn't including it as part of standard designs if we're removing it.

Reply:
Good thought! I removed the clear button from the [SquareOne Figma starter file](https://www.figma.com/file/Wj2g7HVyBQWYIuxgNKMCxf/SQ1-Main-Patterns?node-id=2949%3A20531) and the [Scope Doc Template](https://docs.google.com/document/d/1JxxR4-4eGU1ea8SBwrue4wc53zCSaDdb4k819TYybuA/edit#heading=h.7l1p9okwoaz8) don't see a clear button in the search form. I [chatted w/ Andrew in the QA channel](https://tribe.slack.com/archives/C2XNN0BE0/p1659641297709529) a bit about this and don't think it's necessary for a11y purposes, especially on a form with just a single input (like search is).

Reply:
I would agree there, Davee. Especially when we get to the mobile versions where search might be it's own "takeover", duplicate "X" buttons become a bit confusing.